bundles/org.openhab.binding.bmwconnecteddrive/README.md

```diff
@@ -0,0 +1,686 @@
+| Charging Mode for Profile          | profile-mode        | String | 
+| Charging Window Start Time         | window-start        | String | 
+| Charging Window End Time           | window-end          | String | 
+| Timer 1: Departure Time            | timer1-departure    | String | 
+| Timer 1: Scheduled Days            | timer1-days         | String | 
+| Timer 1: Enabled                   | timer1-enabled      | Switch | 
+| Timer 2: Departure Time            | timer2-departure    | String | 
+| Timer 2: Scheduled Days            | timer2-days         | String | 
+| Timer 2: Enabled                   | timer2-enabled      | Switch | 
+| Timer 3: Departure Time            | timer3-departure    | String | 
+| Timer 3: Scheduled Days            | timer3-days         | String | 
+| Timer 3: Enabled                   | timer3-enabled      | Switch | 
+
+
+#### Location
+
+GPS location and heading of the vehicle.
+
+* Channel Group ID is **location**
+* Available for all vehicles with built-in GPS sensor. Function can be enabled/disabled in the head unit
+* Read-only values
+
+| Channel Label   | Channel ID          | Type         | 
+|-----------------|---------------------|--------------|
+| GPS Coordinates | gps                 | Location     | 
+| Heading         | heading             | Number:Angle | 
+
+#### Last Trip
+
+Statistic values of duration, distance and consumption of the last trip.
+
+* Channel Group ID is **last-trip**
+* Available if *Statistics* is present in *Services Supported*. See [Vehicle Properties](#properties) for further details
+* Read-only values
+
+| Channel Label                           | Channel ID                   | Type          |
+|-----------------------------------------|------------------------------|---------------|
+| Last Trip Date                          | date                         | DateTime      |
+| Last Trip Duration                      | duration                     | Number:Time   |
+| Average Power Consumption per 100 km    | avg-consumption              | Number:Power  |
+| Average Combined Consumption per 100 km | avg-combined-consumption     | Number:Volume |
+| Average Power Recuperation per 100 km   | avg-recuperation             | Number:Power  |
+| Last Trip Distance                      | distance                     | Number:Length |
+| Distance since Last Charge              | distance-since-charging      | Number:Length |
+
+
+#### Lifetime Statistics
+
+Providing lifetime consumption values.
+
+* Channel Group ID is **lifetime**
+* Available if *Statistics* is present in *Services Supported*. See [Vehicle Properties](#properties) for further details
+* Read-only values
+
+| Channel Label                           | Channel ID                   | Type          | 
+|-----------------------------------------|------------------------------|---------------|
+| Average Power Consumption per 100 km    | avg-consumption              | Number:Power  |
```

Comment:
Is the "per 100 km" also applicable if the user lives in an imperial country?

Reply:
Yes, the API is delivering only this value. In the car you can change the display to _km / kwh_ or _mi / kwh_ but this doesn't effect the API content. 
But this is a good point I've overseen so far. I'll perform the conversion inside the binding in order to give consistent feedback in case of Imperial configuration